__tests__/components/Subjects/AddSubject.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import AddSubject from '../../../src/components/Subjects/AddSubject'
+
+const setup = (setupProps = {}) => {
+  const defaultProps = {
+    subjectsType: 'primary',
+    onAdd: jest.fn()
+  }
+  const props = { ...defaultProps, ...setupProps }
+  const wrapper = shallow(<AddSubject {...props} />)
+
+  return {
+    props,
+    wrapper
+  }
+}
+
+describe('AddSubject component', () => {
+  test('should render', () => {
+    const { wrapper } = setup()
+
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  test('should render green button when subjectsType is lingual', () => {
+    const { wrapper } = setup({ subjectsType: 'lingual' })
+
+    expect(wrapper.is('.btn btn-outline-success')).toEqual(true)
```

Comment:
This kind of stuff can be handled via snapshot

Reply:
But is that enough? With this test there's more documentation in tests and... I don't need to look if everything looks good because it's written down in test results. I don't know if additional 'design tests' is something at least accepted.

Reply:
Well, you need to evaluate if testing given part of the application is really giving you more benefits than spending time on writing (and maintaining!) specific test.

The worst that could happen here, would be button having wrong color. I would say it's not worth the efford in this case. But as you are not constrained by deadlines and this app is also for learning purpose, you can play with it a bit more. :)